### PR TITLE
Full Czech translation

### PR DESCRIPTION
**Pull request by Malanius**
### Contains

- Completely translated menu into Czech and fixed some incorrect language use from previous translation
- "Experimental" label in player settings is now translatable
- Changed Behaviour Tree Editor font to NotoSans-Regular
- Added Malanius to credits

### How to test

Start the game, change the language to Czech. See all translations.

### Outstanding before merging

Probably nothing :)

### CODE REVIEW

**Comment by GooeyHub** [comment]
Can one of the admins please verify this patch?

File: engine/src/main/resources/assets/skins/behaviorEditor.skin [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
     "inherit": "mainMenu",
     "text-shadowed": true,
-    "font": "default",
+    "font": "Noto",
```
Comment (msteiger) [inline engine/src/main/resources/assets/skins/behaviorEditor.skin]: I think that this should be `"font": "NotoSans-Regular"` instead.

**Comment by msteiger** [comment]
Looks good to me.

**Comment by Cervator** [comment]
Nice work! :-)

Triggering our bot so we can have a look sometime

@GooeyHub: ok to test

**Comment by GooeyHub** [comment]
Hooray Jenkins reported success with all tests good!

File: engine/src/main/resources/assets/skins/behaviorEditor.skin (hunk 1 above)
Comment (Malanius) [inline engine/src/main/resources/assets/skins/behaviorEditor.skin]: I was wondering if it should be some sans version as it is more similar to the "default" font, I'll update it.

**Comment by GooeyHub** [comment]
Hooray Jenkins reported success with all tests good!

**Comment by Cervator** [comment]
Tested this out locally - looks good, thanks! :-)

Since there are quite a few commits I'm going to do a squash merge to keep history a bit more tidy

**Comment by Malanius** [comment]
Thanks!
Time to celebrate :)

**Pull request merged**